### ISSUE TITLE
Issue #188 exists again

### ISSUE THREAD

**Issue by hlange** [ScorpioBroker/ScorpioBroker#260]
Hello,
I had some problems with the modifiedAt parameter at was looking at the closed tickets and I found the ticket #188, when I tried this I could reproduce the error mentioned there.

![grafik](https://user-images.githubusercontent.com/12150189/141941996-a41e08ba-7f47-47be-8143-b73a6b9161c6.png)

As image I use scorpiobroker/scorpio:scorpio-aaio_latest.

**Comment by amitrnec** [comment]
Hi,
As I have tested this issue after taking the code from dev branch. It is working as expected but i will check also through  scorpiobroker/scorpio:scorpio-aaio_latest image.

Thanks,
Amit.

**Comment by ScorpioBroker** [comment]
Hi we couldn't recreate this bug at the moment. Could you check which containerid your build has with \"docker images\"

**Comment by hlange** [comment]
Hi, that is the version that was fetched with docker compose.
![grafik](https://user-images.githubusercontent.com/12150189/142984965-76e47d7c-5fb2-449c-b3b1-485e7dc9eb8a.png)

The full output:
![grafik](https://user-images.githubusercontent.com/12150189/142985612-52045b67-825e-4a72-ba0f-b081af6d70a6.png)

**Comment by hlange** [comment]
Ok. I tested with the new image, but I still didn't get the right result.

```
henrik@henrik-HCAR357-MI:~$ curl --location --request POST 'http://localhost:9090/ngsi-ld/v1/entities/' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
        "id": "urn:ngsi-ld:Vehicle:A100",
        "type": "Vehicle",
        "brandName": {
                "type": "Property",
                "value": "Mercedes"
        },
        "speed": {
                "type": "Property",
                "value": 80
        }
}'
henrik@henrik-HCAR357-MI:~$ curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/entities/?type=Vehicle&options=sysAttrs' \
--header 'Accept: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'

[{
  "id" : "urn:ngsi-ld:Vehicle:A100",
  "type" : "Vehicle",
  "createdAt" : "2021-11-23T07:51:01.980242Z",
  "modifiedAt" : "2021-11-23T07:51:01.980242Z",
  "brandName" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : "Mercedes",
    "modifiedAt" : "2021-11-23T07:51:01.980242Z"
  },
  "speed" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : 80,
    "modifiedAt" : "2021-11-23T07:51:01.980242Z"
  }
}]henrik@henrik-HCAR357-MI:~curl --location --request PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A100/attrs/speed' \ \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
    "value": "120",
    "type": "Property"
}'
henrik@henrik-HCAR357-MI:~$ curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/entities/?type=Vehicle&options=sysAttrs' \
--header 'Accept: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'
[{
  "id" : "urn:ngsi-ld:Vehicle:A100",
  "type" : "Vehicle",
  "createdAt" : "2021-11-23T07:51:01.980242Z",
  "modifiedAt" : "2021-11-23T07:51:26.973902Z",
  "brandName" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : "Mercedes",
    "modifiedAt" : "2021-11-23T07:51:01.980242Z"
  },
  "speed" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : "120",
    "modifiedAt" : "2021-11-23T07:51:26.973902Z"
  }
}]henrik@henrik-HCAR357-MI:~$ curl --location --request PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A100/attrs/speed' --header 'Content-Type: application/json' --header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' --data-raw '{
    "value": "130",
    "type": "Property"
}'
henrik@henrik-HCAR357-MI:~$ curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/entities/?type=Vehicle&options=sysAttrs' --header 'Accept: application/json' --header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'
[{
  "id" : "urn:ngsi-ld:Vehicle:A100",
  "type" : "Vehicle",
  "createdAt" : "2021-11-23T07:51:01.980242Z",
  "modifiedAt" : "2021-11-23T07:52:04.035370Z",
  "brandName" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : "Mercedes",
    "modifiedAt" : "2021-11-23T07:51:01.980242Z"
  },
  "speed" : {
    "type" : "Property",
    "createdAt" : "2021-11-23T07:51:01.980242Z",
    "value" : "130",
    "modifiedAt" : "2021-11-23T07:51:26.973902Z"
  }
henrik@henrik-HCAR357-MI:~$
```

As you can see, when I update the element twice, the modifiedAt element is not changing.

![grafik](https://user-images.githubusercontent.com/12150189/142988168-3be645fc-11d3-44e1-8297-7dd05b9d6e46.png)

**Comment by amitrnec** [comment]
Hi,
Fixed this issue in #266
